src/Files/File.php

```diff
@@ -1304,8 +1304,15 @@ public function getDeclarationName($stackPtr)
             return $this->tokens[$stackPtr]['content'];
         }
 
+        $stopPoint = $this->numTokens;
+        if ($tokenCode === T_FUNCTION && isset($this->tokens[$stackPtr]['parenthesis_opener']) === true) {
```

Comment:
Is `$tokenCode === T_FUNCTION` important here? The tests pass with & without it.
```suggestion
        if (isset($this->tokens[$stackPtr]['parenthesis_opener']) === true) {
```

Reply:
Good question. The `parenthesis_opener` condition should only be able to pass for `T_FUNCTION` tokens, so I suppose the `T_FUNCTION` check isn't necessary and could be replaced with a comment.